x-pack/plugins/ml/public/components/field_title_bar/field_title_bar.js

```diff
@@ -31,11 +32,22 @@ export function FieldTitleBar({ card }) {
     defaultMessage: 'document count'
   });
 
+  const cardTitleAriaLabel = [fieldName];
```

Comment:
rather than adding this additional check here, could the initialisation of `cardTitleAriaLabel ` be combined with that of `classNames ` on line 23?

Reply:
@jgowdyelastic , that's right, thanks!